community/modules/scripts/spack/scripts/install_spack_deps.yml

```diff
@@ -0,0 +1,38 @@
+# Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+---
+
+- name: Install dependencies for spack installation
+  become: yes
+  hosts: localhost
+  tasks:
+  - name: Install pip3 and git
+    ansible.builtin.package:
+      name:
+      - python3-pip
```

Comment:
This is a requirement for Ansible itself, you can assume it is installed by the toolkit, along with a few other things and virtual env: https://github.com/GoogleCloudPlatform/hpc-toolkit/tree/main/modules/scripts/startup-script#runner-dependencies

Reply:
also, I may be missing something, but I can't really figure where install_spack_deps is being called

Reply:
I can get rid of the python deps, thanks for pointing that out.

As far as where this is called from, I added it to maintain the same functionality as the old spack module. Generally this is added into startup scripts, as: https://github.com/GoogleCloudPlatform/hpc-toolkit/blob/main/tools/validate_configs/test_configs/spack-buildcache.yaml#L65-L66

This also allows the deps to be installed on other VMs than the one running the installer that might want to use spack.